fiesta/apps/accounts/templatetags/user_profile.py

```diff
@@ -19,10 +20,14 @@ def get_user_picture(user: User | None):
     return profile.picture
 
 
-@register.simple_tag(takes_context=True)
-def compute_profile_fullness(context: dict, profile: UserProfile) -> float:
-    # req: HttpRequest = context.get("request")
+@register.simple_tag
+def compute_profile_fullness(user: User) -> float:
+    fields = UserProfileForm().get_form_fields(user)  # Get all field names of UserProfile
```

Comment:
it's classmethod, no need to make an instance for calling the `get_form_fields`

```suggestion
    fields = UserProfileForm.get_form_fields(user)  # Get all field names of UserProfile
```

Reply:
done